app/assets/javascripts/bid.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function() {
+
+  $('.form .bid-now-button').click(function(e) {
+    e.preventDefault();
+    $('.buy-now-button').hide();
+    $('.bid-form').addClass("styled");
+    $('.bid-now-button').addClass("styled-bid-button");
+
+    $(this).unbind("click");
+    $('.bid-now-button').click(function() {
```

Comment:
You can take this `.click` out since it's not doing anything, just unbinding is enough.